_docs/services/app-autoscaler.md

```diff
@@ -0,0 +1,472 @@
+---
+parent: services
+layout: docs
+sidenav: true
+title: Application AutoScaler
+name: "app-autoscaler"
+description: "Cloud Foundry App-AutoScaler: scale number of application instances based on metrics"
+status: "Beta"
+---
+
+We are pleased to introduce App-AutoScaler to the platform. [Cloud.gov](https://www.cloud.gov/) has always made it easy to handle increased traffic by scaling your application memory or instances with a single command. With App-Autoscaler, responding to changes in traffic and resource usage is now even easier.
```

Comment:
I have mixed feelings about using an abbreviation like `App-Autoscaler` in the content, but we're not consistent at all about this on our website. And I can tell that this is the name for the service in general and you're defining it in this paragraph. So I'm good with it.

Reply:
I had like 5 variations of App + Auto + Scaler depending on where I was getting the information from.

I standardized on `App-AutoScaler` since that was what the maintainers called it in the main README in the release.

The only kind of made up terminology is `Scaling Policy`.  Most of the upstream documentation just calls this "the policy" but I artificially standardized on Scaling Policy for clarity/consistency